src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //равнобедренная трапеция
+    private double height; // высота
+
+    public IsoscelesTrapezoid(String color, double firsSide, double secondSide, double height) {
```

Comment:
There is a typo in the constructor parameter name `firsSide`. It should be `firstSide` to match the class property name.